### DISCUSSION TITLE
Data Model ERD

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#5735] [Feature Requests]
In addition to, or instead of, the Data Model's Table Layout, we could have an ERD view that shows all Collections... and perhaps Fields too. This would be useful for visualizing your data architecture, specifically your relationships. Also, if relationships are clickable, then this acts as a navigation for managing relations. Below are a few options and inspirations for those visualizations:

* https://cdn.discordapp.com/attachments/725391741376659549/840251050274455653/unknown.png
* https://gojs.net/latest/samples/entityRelationship.html
* https://gojs.net/latest/samples/stateChart.html

**Comment by rokdd** [comment]
Just because I found the following blog post (and I guess benhaynes knows already):

[Community Spotlight: ERD Generator Extension for Visualizing Data Relationships](https://directus.io/blog/community-spotlight-erd-generator-extension-for-visualizing-data-relationships/)

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
